### PR TITLE
added flexbox

### PR DESCRIPTION
**Pull request by PerryL17**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,6 +1,22 @@
 
 @media screen and (max-width: 736px) {
 
+}
+nav ul {
+    display: flex;
+    justify-content: space-between;
+    list-style-type: none;
+    position: fixed;
```
Comment (ckesurf) [inline css/index.css:16]: Nice, you got the header to be sticky!

**Review by ckesurf — COMMENTED** [review]
Nitpicky comments - usually not a big deal for private personal sites, but if this portfolio is going to end up in front of recruiters and hiring managers, I figured it's worthwhile to mention these:
### index.html
Typo on line 23: `perprtual` -> `perpetual`
Line 25: `tranistioning` -> `transitioning`
Line 34: `tradtional` -> `traditional`
Line 37: `developement` -> `development`
Line 38: `Code Tenderlion` -> `Code Tenderloin`, 
also `totally` -> `totaling`

File: index.html [hunk 2]
```diff
@@ -3,10 +3,19 @@
...
     <link rel="stylesheet" href="css/index.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
-    <img src="IMG_0891.jpg" width="300" height="500" />
+   
     <title>About Perry Littrell</title>
+    <nav class="menu">
+      <ul>
+        <li><a href="#about">About</a></li>
+        <li><a href="#experience">Experience</a></li>
+        <li><a href="#skills">Skills</a></li>
```
Comment (ckesurf) [inline index.html]: I noticed while looking at your website, that some of these nav links (`about` and `experience`) go to the `<p>` element, while `skills` go to the header `<h2>` of the section. Specifically, some of these ids are in the `<p>` element, which is awkward navigation-wise, while the `skills` id goes to the header element, which is more what I would expect. Could we move the id from the `<p>` element to the `<section>` element, for consistency?

File: index.html [hunk 3]
```diff
@@ -44,9 +53,9 @@ <h2>Connect:</h2>
         <li>https://www.instagram.com/gemcrate/</li>
       </ul>
     </section>
-    <section>
+    <section id="leave_message">
```
Comment (ckesurf) [inline index.html]: ^Yeah like this is perfect, you have the id in the section element, which would make sense to me

**Review by ckesurf — CHANGES_REQUESTED** [review]
One of the requirements for the assignment was to make experience a grid instead of a list:

>  Change the layout of your "Experience" section using rows and columns so items are displayed in a grid rather than a list

Right now there's just a single paragraph element; could we add more items, and could we turn that section into a grid?

File: css/index.css [hunk 4]
```diff
@@ -27,14 +43,26 @@ h2 {
     padding: 1.5rem;
     background-color: lightgray;
 }
+#leave_message {
+    display: flex;
+    border: 1px solid #000;
+    background: cornsilk;
+    
+}
+.form {
```
Comment (ckesurf) [inline css/index.css]: I don't know if you need a new class, `.form`, I would either:
- Change the styling for `form` element directly in index.css, like you already do with `nav` and other HTML elements
```
form {
   ....
}
```

- or, move the styling into the `#leave_message` which is what you have above.

File: index.html (hunk 2 above)
Comment (PerryL17) [reply index.html]: hi C! thank you for the feedback. I'm trying to work through it now.

File: css/index.css (hunk 4 above)
Comment (PerryL17) [reply css/index.css]: ok, did that! much cleaner

**Comment by PerryL17** [comment]
About the experience list, I now see that in order to finish the assignment
I need to convert it to a list in some way and then create a grid. I'm a
bit lost on the grid part. I see that there was an optional css grid course
that I wasn't able to get through last week. I will see if I can find
sometime this weekend. Thank you for taking the time to help. Last note,
why doesn't VS code have spell check?!?!?! ;)

**Pull request merged**